app/inputs/percentage_input.rb

```diff
@@ -0,0 +1,24 @@
+class PercentageInput < SimpleForm::Inputs::NumericInput
```

Comment:
Made a custom simple_form input to pull the different markup out of the view file, and have enough control to have the styles we need be moderately less cumbersome & fragile.